app/views/tasks/_task_table.html.erb

```diff
@@ -0,0 +1,60 @@
+<table>
+  <tr>
+    <th>
+      Name
+    </th>
+    <th>
+      Description
+    </th>
+    <th>
+      Owner
+    </th>
+    <th>
+      Status
+    </th>
+    <% if @task.completed_date.present? %>
+      <th>Completion Date</th>
+    <% end %>
+    <% if current_page?(action: 'show') %>
```

Comment:
What do you think about passing in a variable to the partial from the calling page to set this value. Then the calling page can be responsible for the language and this partial doesn't have to have knowledge of what actions it may be used for.

So here you would do something like:

```erb
      <th class="center">
        <%=  action_label %>
      </th>
```

And in the `show.html.haml` view, for example, you could use:

```erb
<%= render "task_table" action_label: "Edit" %>
```